contracts/Operations.sol

```diff
@@ -276,38 +276,94 @@ contract Operations is OperationsFace {
 	}
 
 	function checkProxy(bytes32 _txid) internal when_proxy_confirmed(_txid) returns (uint txSuccess) {
-		var tx = proxy[_txid];
-		var success = tx.to.call.value(tx.value).gas(tx.gas)(tx.data);
+		var txn = proxy[_txid];
+		// solium-disable-next-line security/no-call-value
+		var success = txn.to.call.value(txn.value).gas(txn.gas)(txn.data);
 		TransactionRelayed(_txid, success);
 		txSuccess = success ? 2 : 1;
 		delete proxy[_txid];
 	}
 
 	// Modifiers
 
-	modifier only_owner { if (grandOwner != msg.sender) throw; _; }
-	modifier only_client_owner { var newClient = clientOwner[msg.sender]; if (newClient == 0) throw; _; }
-	modifier only_required_client_owner { var newClient = clientOwner[msg.sender]; if (!client[newClient].required) throw; _; }
-	modifier only_ratified{ if (!fork[proposedFork].ratified) throw; _; }
-	modifier only_unratified { if (!fork[proposedFork].ratified) throw; _; }
+	modifier only_owner {
+		require(grandOwner == msg.sender);
+		_;
+	}
+
+	modifier only_client_owner {
+		var newClient = clientOwner[msg.sender];
+		require(newClient != 0);
+		_;
+	}
+
+	modifier only_required_client_owner {
+		var newClient = clientOwner[msg.sender];
+		require(client[newClient].required);
+		_;
+	}
+
+	modifier only_ratified {
+		require(fork[proposedFork].ratified);
+		_;
+	}
+
+	modifier only_unratified {
+		require(!fork[proposedFork].ratified);
```

Comment:
Seems you've actually fixed a bug here :) The original code does the same as `only_ratified`